### PR TITLE
issue #8515 Spelling mistakes found checking `en-GB` to `en-US`

### PR DESCRIPTION
**Pull request by albert-github**
Spelling corrections, `en-GB` -> `en-US`

### CODE REVIEW

File: Nef_2/include/CGAL/Nef_2/PM_point_locator.h [hunk 1]
```diff
@@ -536,7 +536,7 @@ class PM_point_locator : public
   The efficiency of this point location module is mostly based on
   heuristics. Therefore worst case bounds are not very expressive. The
   query operations take up to linear time for subsequent query
-  operations though they are better in practise. They trigger a one-time
+  operations though they are better in practize. They trigger a one-time
```
Comment (afabri) [inline Nef_2/include/CGAL/Nef_2/PM_point_locator.h:539]:
```suggestion
  operations though they are better in practice. They trigger a one-time
```

File: Number_types/include/CGAL/Sqrt_extension/Algebraic_extension_traits.h [hunk 2]
```diff
@@ -22,7 +22,7 @@ template <class COEFF, class ROOT, class ACDE_TAG,class FP_TAG>
 class Algebraic_extension_traits<CGAL::Sqrt_extension<COEFF,ROOT,ACDE_TAG,FP_TAG> > {
 /* needed to 'add up' sqrt_extensions in iterator range such that all roots
    are collected in order to keep operation time minimal all scalar coeffs
-   are set to 1 by standardise.
+   are set to 1 by standardize.
```
Comment (afabri) [inline Number_types/include/CGAL/Sqrt_extension/Algebraic_extension_traits.h:25]: looks strange.  Maybe "by default"?

**Review by afabri — APPROVED** [review]
Thanks a lot, Albert

File: Number_types/include/CGAL/Sqrt_extension/Algebraic_extension_traits.h (hunk 2 above)
Comment (afabri) [reply Number_types/include/CGAL/Sqrt_extension/Algebraic_extension_traits.h:25]: It is only in a comment so let's just ignore it

**Comment by afabri** [comment]
@sloriot @lrineau I would just merge it.

File: Number_types/include/CGAL/Sqrt_extension/Algebraic_extension_traits.h (hunk 2 above)
Comment (albert-github) [reply Number_types/include/CGAL/Sqrt_extension/Algebraic_extension_traits.h]: Indeed, changed it

**Pull request merged**